README.md

```diff
@@ -20,7 +20,7 @@ by Tingying Peng, Kurt Thorn, Timm Schroeder, Lichao Wang, Fabian J Theis, Carst
 |     [example_brightfield](https://github.com/peng-lab/PyBaSiC-examples/blob/main/example_brightfield.ipynb)     |                                                                                                                                                                                  100 continuous brightfield frames of a time-lapse movie of differentiating mouse hematopoietic stem cells.                                                                                                                                                                                  | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1PPiuT--gamaQBpuUdYMAmtwe5b5-eLJ5?usp=sharing) |
 | [example_timelapse_nanog](https://github.com/peng-lab/PyBaSiC-examples/blob/main/example_timelapse_nanog.ipynb) | 189 continuous fluorescence frames of a time-lapse movie of differentiating mouse embryonic stem cells, which move much more slower compared to the fast moving hematopoietic stem cells, resulting in a much larger correlation between frames. Note that in this challenging case, the automatic parameters are no longer optimal, so we use the manual parameter setting (larger smooth regularization on both flat-field and dark-field) to improve BaSiC’s performance. | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1rLmkGH5Zb_kWgUZVksgt-XR3jhlWMWei?usp=sharing) |
 
-You can also find examples of running the package at [folder Notebooks](https://github.com/peng-lab/PyBaSiC/tree/main/Notebooks). Data used in the examples and a description can be downloaded from [Dropbox](https://www.dropbox.com/s/plznvzdjglrse3h/Demoexamples.zip?dl=0)
+You can also find examples of running the package at [folder Notebooks](https://github.com/peng-lab/PyBaSiC/tree/main/Notebooks). Data used in the examples and a description can be downloaded from [Dropbox](https://www.dropbox.com/s/plznvzdjglrse3h/Demoexamples.zip?dl=0) or [Zenodo](https://zenodo.org/record/6334810#.YiYgWHrMJro).
```

Comment:
Thanks for adding data in Zenodo! The DOI URL is supposed to be more stable.
https://doi.org/10.5281/zenodo.6334810
Can you change it to this URL?